### PR TITLE
2.1.1 GA PR 

### PR DESCRIPTION
**Pull request by cmastr**
Preview:

http://docs-d2iq-com-pr-3984.s3-website-us-west-2.amazonaws.com/ 

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cneth — APPROVED** [review]

**Comment by msdolbey** [comment]
Making a quick change to the formatting of one page

**Comment by msdolbey** [comment]
Those linting errors should be fine after merge - they look good to me (they're actually in that accept.txt file, must just have not been checked at the same build or against the main branch or whatever, not sure)

**Review by msdolbey — APPROVED** [review]
👍 
Actually is perfect

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    spec:
+      interval: 1m0s
+      ref:
+        tag: v2.1.1
+      timeout: 20s
+      url: https://github.com/mesosphere/dkp-catalog-applications  
+    EOF
+    ```
+
+1.  Verify that you can see the workspace catalog `Apps` available in the UI and in the CLI, using `kubectl`:
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:42]: I think this was my initial change anyway - but I think a comma here shows that they are two separate actions that do the same thing. So just committing here.

```suggestion
1.  Verify that you can see the workspace catalog `Apps` available in the UI, and in the CLI, using `kubectl`:
```

**Review by cwyl02 — APPROVED** [review]

**Pull request merged**